### ISSUE TITLE
Rosetta error on MacOS 13.6.2

### ISSUE THREAD

**Issue by adaminato42** [abiosoft/colima#926]
### Description

Trying to start any x86/arm64 VMs results in a Rosetta error:

```
docker run -it --rm --platform linux/amd64 debian:buster-slim
rosetta error: unhandled auxillary vector type 28
```

This was reported in https://github.com/abiosoft/colima/issues/855 but that issue has since been closed without any resolution.

I have run both `colima delete` and `lima delete` before restarting colima using the folliowing command:
```
colima start --arch aarch64 --vm-type=vz --vz-rosetta --memory 4
```

### Version

colima version && limactl --version && qemu-img --version.

colima version HEAD-ba1be00
git commit: ba1be00e9aec47f2c1ffdacfb7e428e465f0b58a

runtime: docker
arch: aarch64
client: v24.0.7
server: v24.0.7
limactl version 0.19.0

qemu-img version 8.1.3
Copyright (c) 2003-2023 Fabrice Bellard and the QEMU Project developers

### Operating System

- [ ] macOS Intel <= 13 (Ventura)
- [ ] macOS Intel >= 14 (Sonoma)
- [X] Apple Silicon <= 13 (Ventura)
- [ ] Apple Silicon >= 14 (Sonoma)
- [ ] Linux

### Output of `colima status`

colima status
INFO[0000] colima is running using macOS Virtualization.Framework
INFO[0000] arch: aarch64
INFO[0000] runtime: docker
INFO[0000] mountType: virtiofs
INFO[0000] socket: unix:///Users/adrian.d/.colima/default/docker.sock

### Reproduction Steps

1. colima delete
2. lima delete
3. colima start --arch aarch64 --vm-type=vz --vz-rosetta --memory 4
4. docker run -it --rm --platform linux/amd64 debian:buster-slim

### Expected behaviour

Expected behaviour is for amd64 VM to start.  Running for arm64 works as expectged

```
docker run -it --platform linux/arm64/v8 --rm debian:buster-slim
Unable to find image 'debian:buster-slim' locally
buster-slim: Pulling from library/debian
78b429e3efdd: Pull complete
Digest: sha256:8e5da7fd9aef8c0b3f23bb8c3f391667b50929984b7daa666ea1c7730f3b73b9
Status: Downloaded newer image for debian:buster-slim
root@dfdefdc47381:/#
```

### Additional context

We are unable to upgrade to MacOS 14.x at this time due to corporate policies.

**Comment by adaminato42** [comment]
I have confirmed this issue still persists with MacOS 13.6.3

**Comment by fabianokoseki** [comment]
I also have the same issue. Decided to downgrade to the 0.5.6 version until this is fixed, to avoid upgrading to MacOS 14.

Here is how I did the downgrade: https://github.com/abiosoft/colima/issues/877

**Comment by joshua-paypay** [comment]
I had the same issue in MacOS 13.6. Upgrading 14.3 solved my issue.

```
colima version 0.6.7
git commit: ba1be00e9aec47f2c1ffdacfb7e428e465f0b58a

runtime: docker
arch: aarch64
client: v24.0.7
server: v24.0.7
```

**Comment by artygus** [comment]
There's a bit of explanation here https://github.com/lima-vm/lima/discussions/2774#discussioncomment-11007396

TL;DR there was a [change](https://github.com/torvalds/linux/commit/317c8194e6aeb8b3b573ad139fc2a0635856498e) in linux kernel since version 6.3 that rosetta on Ventura can't handle. It was fixed in Sonoma, suggested  possible workaround with the preserve flag doesn't really work. I think the only option for colima to support is to either create a custom kernel build or use ubuntu jammy cloud image (22.04 LTS) for example which is on kernel 5.
